controller.go

```diff
@@ -126,6 +117,19 @@ func indexHandler(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintln(w, "Home page")
 }
 
+// Get an int form value from the request
```

Comment:
nit: let's make clear that `default` is only used if the field name does not exist, if it exists and it's an invalid value an error will be returned.

Reply:
Done